OrbitCore/SymbolsClient.cpp

```diff
@@ -84,11 +88,32 @@ void SymbolsClient::HandleResponse(const Message& message, uint64_t id) {
   core_app_->OnRemoteModuleDebugInfo(infos);
 
   // Finalize transaction.
-  id_sessions_mutex_.Lock();
+  mutex_.Lock();
   std::shared_ptr<Session> session = id_sessions_[id];
   id_sessions_.erase(id);
-  id_sessions_mutex_.Unlock();
+
+  std::vector<std::string> not_found_module_names =
+      std::move(id_not_found_module_names_[id]);
+  id_not_found_module_names_.erase(id);
+  mutex_.Unlock();
+
   if (session != nullptr) {
     core_app_->ApplySession(*session);
   }
+
+  // Show an error MessageBox for modules not found locally nor remotely.
+  for (const ModuleDebugInfo& module_info : infos) {
+    if (!module_info.m_Functions.empty()) {
+      not_found_module_names.erase(std::find(not_found_module_names.begin(),
+                                             not_found_module_names.end(),
+                                             module_info.m_Name));
+    }
+  }
+  if (!not_found_module_names.empty()) {
+    std::string text = "Could not load symbols for modules:";
+    for (const std::string& module_name : not_found_module_names) {
+      text += "\n  " + module_name;
+    }
```

Comment:
absl::StrJoin?

Reply:
Mmh, this would be it:
```
std::string text = "Could not load symbols for modules:\n  " +
                    absl::StrJoin(not_found_module_names, "\n  ");
```
The separator `"\n "` would have to appear twice (I also need it at the beginning), which to me is less readable.

Reply:
I think this is fine.. for loop does not make things better in my opinion.

Reply:
Done.